pkg/block/local/walker.go

```diff
@@ -81,7 +83,7 @@ func (l *LocalWalker) Walk(_ context.Context, storageURI *url.URL, options WalkO
 			}
 			const dirPerm = 0o755
 			_ = os.MkdirAll(l.cacheLocation, dirPerm)
-			cacheName := filepath.Join(l.cacheLocation, nanoid.Must()+"-import.json")
+			cacheName := filepath.Join(l.cacheLocation, gonanoid.Must()+"-import.json")
```

Comment:
I don't understand.  Shouldn't this live under the system temp file area, and be handled that way?

Reply:
This might be quite surprising or confusing to users.  We should take care to document this.